src/runtime/replay.py

```diff
@@ -18,7 +18,6 @@
 def replay_core(replay_job: Job, training_initial_job: Job) -> list:
     """The function create a set with timestamps of events, then create a list of requests
         simulating the log in the time passing
-
```

Comment:
this should be left as it was.. the documentation follows this format https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html